roles/openshift_sdn/files/sdn-ovs.yaml

```diff
@@ -29,6 +29,7 @@ spec:
       serviceAccountName: sdn
       hostNetwork: true
       hostPID: true
+      priorityClassName: system-cluster-critical
```

Comment:
I think most of them should be node critical, as it it the highest priority in the cluster.

Reply:
> I think most of them should be node critical, as it it the highest priority in the cluster.

why?

Reply:
There is a chance that system-node-critical pods will evict them system-cluster-critical, rare but possible.

Reply:
I kind of agree, (though now I see there name is a bit misleading, atleast to me, it seemed like cluster-critical is higher than node-critical, but node-critical is 1000+ of cluster critical) so I think I am planning for following:

descheduler: system-cluster-critical (it is ok if it is moved by system-node-critical) 
syn, sdn,sdn-ovs: system-node-critical  (as these will be on each node irrespective of their types)
fluentd: system-cluster-critical (in my understanding, I think, it's ok to keep them at system-cluster-critical)

Reply:
I think fluentd should be at node-critical as this should not be evicted at all. If evicted, we will not have logging from the node.

Reply:
> I think fluentd should be at node-critical as this should not be evicted at all. If evicted, we will not have logging from the node.

What  do you think should have higher priority between "sdn,sdn-ovs,sync" and fluentd?

Reply:
While I agree with you, I think sizing of node should be done based on all the system-node critical and system-cluster critical pod. If node cannot hold all the pods then only, we should think of priority between "sdn,sdn-ovs,sync" and fluentd.

Reply:
> While I agree with you, I think sizing of node should be done based on all the system-node critical and system-cluster critical pod. If node cannot hold all the pods then only, we should think of priority between "sdn,sdn-ovs,sync" and fluentd.

If the sizing is like that the node can afford all "sdn,sdn-ovs,sync" and fluentd, why to worry about between cluster and node critical priorities?

Reply:
True as I told earlier, the chances of eviction are very less but this to be on safe side :)

Reply:
Is there outstanding debate here?